src/index.ts

```diff
@@ -20,6 +20,12 @@ const mappers: Record<CrowdinEvents, (num: number) => string> = {
   },
 }
 
+const colors: {[event: string]:number} = {
```

Comment:
This part should be properly typed
```suggestion
const colors: Record<CrowdinEvent, number> = {
```

Reply:
Fixed it